src/systems/ec/implementations/react/templates/pages/event-detail/examples/Default.jsx

```diff
@@ -0,0 +1,815 @@
+                dapibus eleifend. Morbi quis eros diam. Phasellus vitae pulvinar
+                mauris, ut consequat nibh. Quisque nec condimentum nibh, eu
+                pretium libero. Quisque interdum blandit dolor, eu imperdiet
+                sapien. Morbi tristique malesuada luctus. Nulla vel eleifend
+                libero. Quisque non lacinia tortor. Nunc congue nunc id purus
+                ullamcorper vehicula. Vestibulum fermentum imperdiet urna, eget
+                suscipit ipsum bibendum sed. Quisque in ultrices velit, ut
+                aliquam nisl.
+              </p>
+              <p className="ecl-u-type-paragraph ecl-u-mb-none">
+                Ut convallis est et nisi gravida, quis rhoncus mi tincidunt.
+                Suspendisse maximus efficitur erat, at fermentum nibh vulputate
+                non.
+              </p>
+            </div>
+          </div>
+
+          {/* :'( Override of heading spacing */}
+          <h2 className="ecl-u-type-heading-2 ecl-u-mt-2xl ecl-u-mt-md-3xl ecl-u-mb-l">
+            Contact
+          </h2>
+
+          <div className="ecl-row">
+            <div className="ecl-col-12 ecl-col-md-6">
+              {/* :'( Override of heading spacing */}
+              <h3 className="ecl-u-type-heading-3 ecl-u-mt-none ecl-u-mb-m ecl-u-mb-md-l">
+                General contact
+              </h3>
+
+              <div className="ecl-row ecl-u-type-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Phone number
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  (0)2 29 56186 (Commission switchboard)
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Postal address
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  Directorate-General for Education and Culture European
+                  Commission B-1049 Brussels Belgium
+                </div>
+              </div>
+
+              <div className="ecl-row ecl-u-type-m ecl-u-mt-m">
+                <div className="ecl-col-12 ecl-col-md-4 ecl-u-type-bold">
+                  Email address
+                </div>
+                <div className="ecl-col-12 ecl-col-md-8 ecl-u-mt-2xs">
+                  <Link
+                    label="directorate-general@ec.europa.eu"
+                    href="/example"
+                  />
+                </div>
+              </div>
+            </div>
+
+            <div className="ecl-col-12 ecl-col-md-6 ecl-u-mt-l ecl-u-mt-md-none">
+              <h3 className="ecl-u-type-heading-3 ecl-u-mt-none ecl-u-mb-m ecl-u-mb-md-l">
+                Press contact
+              </h3>
+
+              <div className="ecl-row ecl-u-type-m">
```

Comment:
you should not nest rows

Reply:
why not? We don't really have other options to set consistent column width design